game.rb

```diff
@@ -0,0 +1,50 @@
+require_relative 'item'
+require 'date'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at, :publish_date, :authors
+  attr_reader :id, :title
+
+  def initialize(title, multiplayer, last_played_at, publish_date, authors = [])
+    @id = Random.rand(1..1000)
+    super()
+    @title = title
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+    @publish_date = Date.parse(publish_date).strftime('%Y/%m/%d')
+    @can_be_archived = can_be_archived?
+    @authors = []
+    add_authors(authors)
+  end
+
+  # def self.json_create(object)
+  #   new(object['title'], object['multiplayer'], object['last_played_at'], object['publish_date'], object['authors'])
+  # end
```

Comment:
- [ ] Please remove any unused code for a cleaner code base

Reply:
done